.github/workflows/client.yml

```diff
@@ -25,20 +24,18 @@ jobs:
       - name: Build Go
         run: go build ./...
 
-      - name: Create test results directory
-        run: mkdir test-results
-
       - name: Install gotestsum
         run: go get gotest.tools/gotestsum
 
       - name: Run Go tests
-        run: gotestsum --junitfile test-results/unit-tests.xml
+        run: gotestsum
 
-      - name: Publish unit test results
-        uses: EnricoMi/publish-unit-test-result-action@v1
-        if: always()
+  client-scan:
```

Comment:
I'm wondering which name would be better: `client-scan` or `client-gosec`. No strong opinion, so merging it as is. We can always update it in the future.